openrtx/src/ui/ui.c

```diff
@@ -640,6 +645,27 @@ void _ui_changeContrast(int variation)
     display_setContrast(state.settings.contrast);
 }
 
+void _ui_enterStandby()
+{
+    if(standby)
+        return;
```

Comment:
Is this allowed under Allman bracing or should I always use {} even with just a return?

Reply:
Keep it as it is now, we're not super-strict about those things!